uploadNeuroDB/NeuroDB/File.pm

```diff
@@ -509,6 +509,27 @@ sub removeWhitespace {
         return @vars;
     }
 }
+
+=pod
+
+=head3 filterParameters
+
+Manipulates the NeuroDB::File object's parameters and removes all parameters of
+length > 1000
+
+=cut
+sub filterParameters {
+    my $this = shift;
+
+    my $parametersRef = $this->getParameters();
+
+    foreach my $key (keys %{$parametersRef}) {
```

Comment:
I would avoid hardcoding `1000`. You could define a constant in `File.pm`, say `MAX_PARAMETER_LENGTH`, and use this everywhere instead of the actual value.

Reply:
@nicolasbrossard Fixed it in the next PR :)